Documentation/guides/messages/xa0125.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Xamarin.Android error XA0125
+description: XA0125 error code
+ms.date: 08/07/2020
+---
+# Xamarin.Android error XA0125
+
+## Issue
+
+This issue happens when you are trying to use fast deployment on a device which
+does not support it. Fast deployment requires features which are not available
+on devices running API 20 or lower. The Fast Deployment system makes use of the
+`run-as` feature of the Android OS. This feature was either not available or had
```

Comment:
I think we could use a link to `run-as`, but I couldn't find a good doc.

Maybe the source code is an OK link, they have decent comments: https://android.googlesource.com/platform/system/core.git/+/refs/heads/master/run-as/run-as.cpp#42